src/App.tsx

```diff
@@ -5,7 +5,6 @@ import { AppRegistry, Dimensions, LogBox, StyleSheet, View } from 'react-native'
 import { Toaster } from 'sonner-native';
 import { MobileWalletProtocolProvider } from '@coinbase/mobile-wallet-protocol-host';
 import { DeeplinkHandler } from '@/components/DeeplinkHandler';
-import { AppStateChangeHandler } from '@/components/AppStateChangeHandler';
```

Comment:
removing `AppStateChangeHandler` as it was being used only for WC v1 subscription when coming back from background